index.html

```diff
@@ -305,6 +305,50 @@ <h5 class="about-me-heading">About Me</h5>
         </ul>
       </div>
     </section>
-    
+    <footer class="footer">
+      <h2 class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xpzbopvv" method="POST">
+        <ul class="contact-section">
+          <li class="form-items">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="name"
+              maxlength="20"
+              placeholder="Full name"
+              required
+            />
+          </li>
+          <li class="form-items">
+            <input class="email" type="email" placeholder="Email address" required />
+          </li>
+          <li class="form-items">
+            <textarea
+              class="textarea"
+              name="textarea"
+              id="textarea"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter your text here"
+              required
+            >
+            </textarea>
+          </li>
+        </ul>
+        <button type="submit" class="submit-button">Get in Touch</button>
```

Comment:
- [ ] Please remove border from this button and use correct paddings and make sure it matches with the Figma design

### Your design

![image](https://user-images.githubusercontent.com/83803414/165702617-a45296b8-18ab-4538-be0c-003d917bfcff.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165702583-cfb6e079-8f87-4951-b75d-aa1c6b17b849.png)